gtk4/src/auto/stack_switcher.rs

```diff
@@ -127,6 +129,7 @@ pub struct StackSwitcherBuilder {
     visible: Option<bool>,
     width_request: Option<i32>,
     accessible_role: Option<AccessibleRole>,
+    orientation: Option<Orientation>,
```

Comment:
This one needs a since annotation as well

Reply:
Hi @bilelmoussaoui , that one might be more tricky. https://gitlab.gnome.org/GNOME/gtk/-/commit/c848a5139537da8f6b90109598e30d6abcf30696 How do you annotate that a class got a new interface in a specific version?

Reply:
You can't currently, and also the `glib::wrapper!` macro can't express that currently

Reply:
I wouldn't bother for now. Just keep a comment on GTK's Gir.toml file on StackSwitcher's entry

Reply:
done